api_docs/changelog.md

```diff
@@ -20,6 +20,10 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 190**
+
+* [`DELETE /realm/emoji/{emoji_name}`](/api/deactivate-custom-emoji): This endpoint now returns `404` status code when an emoji does not exist instead of `400`.
```

Comment:
```suggestion
* [`DELETE /realm/emoji/{emoji_name}`](/api/deactivate-custom-emoji): This endpoint
  now returns an HTTP status code of 404 when an emoji does not exist, instead of 400.
```

Reply:
Also, note the line breaks in this file for other feature level entries.